src/pages/Account/Login/Login.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+
+import { Formik, Form, Field, FastField, ErrorMessage } from 'formik';
+import { Link } from 'react-router-dom';
+
+import login from '@/apis/login';
+import { ERROR_CODES, MESSAGES, STATUS_CODES } from '@/constants/constantValues';
+import { loginSchema } from '@/helpers';
+import useLogin from '@/hooks';
+
+import { ILoginInfo } from './types';
+import { IUser } from '../types';
+
+export default function Login() {
+  const { handleLoginResponse } = useLogin();
+  const [loginInfo, setLoginInfo] = useState<ILoginInfo>({});
+  const [showPassword, setShowPassword] = useState<boolean>(false);
+
+  const handleLogin = (user: IUser, setSubmitting: { (isSubmitting: boolean): void }) => {
+    login(user)
```

Comment:
Investigate whether promises can be replaced with async/await.

Reply:
I will create a new issue for this.